src/Gui/Action.cpp

```diff
@@ -539,6 +539,34 @@ void WorkbenchGroup::addTo(QWidget *w)
         connect(_group, SIGNAL(triggered(QAction*)), box, SLOT(onActivated (QAction*)));
         bar->addWidget(box);
     }
+    else if (w->inherits("QMenuBar")) {
+        auto menuBar = qobject_cast<QMenuBar*>(w);

```

Comment:
You should have `#include <QMenuBar>` then.

Reply:
Fixed